.golangci.yml

```diff
@@ -34,6 +34,10 @@ linters-settings:
     go: "1.16"
 
 issues:
+  exclude-use-default: false # default rules exclude doc comments check :(
```

Comment:
i'd also include a comment pointing to https://github.com/golangci/golangci-lint/issues/456 so it's not lost in the git log if there is ever reshuffling of this file

Reply:
Good point. Done.